<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAdministratorWindow</class>
 <widget class="QMainWindow" name="QAdministratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtParkingPOS - Administrador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::East</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Registro</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Usuario:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userLEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Contraseña:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passLEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Confirmar Contraseña:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="conpassLEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addPButton">
            <property name="text">
             <string>Añadir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Cortes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Mantenimiento</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>548</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>userLEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addPButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passLEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addPButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>conpassLEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addPButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
